index.js

```diff
@@ -0,0 +1,96 @@
+let firstNumber;
```

Comment:
Try to avoid global variables, but better use classes instead. But as far as I remember that was the next thing you planned to learn anyway, so for the current state that's fine.